lib/ConfirmationDropdown/ConfirmationDropdownContent.js

```diff
@@ -51,18 +52,19 @@ const ConfirmationDropdownContent = (
         >
           {confirmationText}
         </Button>
-
-        <Button
-          types={sharedButtonProps}
-          clickHandler={() => {
-            onHide();
-            onCancel();
-            setShowContent(false);
-          }}
-          className="confirmation-dropdown__cancel"
-        >
-          Cancel
-        </Button>
+        {secondaryAction || (
```

Comment:
I thought the cancel button was consistent across all confirmation dropdowns. Secondary actions appeared in another place instead. Has this changed?

Reply:
See https://projects.invisionapp.com/share/W5PL9F9SX4M#/screens/339318859 how we have a secondary action instead of a cancel button, hopefully that clears things up